models/engine/file_storage.py

```diff
@@ -68,3 +68,18 @@ def delete(self, obj=None):
     def close(self):
         """call reload() method for deserializing the JSON file to objects"""
         self.reload()
+
+    def get(self, cls, id):
+        """returns the object based on the class and its ID
+           returns None if not found"""
```

Comment:
I'm not a fan of comments with double quotes, feels like it makes too much noise, but this isn't a problem